api/directory_watcher.py

```diff
@@ -29,15 +29,19 @@ def calculate_hash(user,image_path):
     return hash_md5.hexdigest() + str(user.id)
 import os
```

Comment:
Not sure how that import os got there. I don't think we need it because at the start of the file

Reply:
Dont know either, I dont see it in the files changed(?)